openedx/features/caliper_tracking/transformers/notes_transformers.py

```diff
@@ -0,0 +1,39 @@
+"""
+Transformers for all the navigation events
+"""
+
+import json
+
+
+def edx_course_student_notes_added(current_event, caliper_event):
+    """
+    The server emits an edx.course.student_notes.added event when the student's
+
+
+    :param current_event: default log
+    :param caliper_event: log containing both basic and default attribute
+    :return: final created log
+    """
+    caliper_event.update(
```

Comment:
```suggestion
    caliper_event.update({
    'type': 'AnnotationEvent',
    'action': 'Bookmarked',
    'object': {
        'id': current_event['referer'],
        'type': 'Annotation',
        'extensions': json.loads(current_event['event'])
    }
})
```

Reply:
`course_id` is more relevant to the `object` dict as it is giving information where the note is being added, that's why i think it would be more appropriate to place `course_id` in `object.extensions` rather than `extensions.extra_fields`

Reply:
I agree with @HamzaFarooqArbisoft here.